molgenis-security/src/main/java/org/molgenis/security/account/PasswordResetTokenRepositoryImpl.java

```diff
@@ -37,7 +37,7 @@
   @Transactional
   @Override
   public String createToken(User user) {
-    if (!user.isActive()) {
+    if (!Boolean.TRUE.equals(user.isActive())) {
```

Comment:
```User.isActive()``` should return a boolean instead of Boolean since UserMetadata states that active is required.